packages/test-app/app/components/chart-bar-example.hbs

```diff
@@ -77,6 +85,17 @@
   @valueAxisScale="shared"
 />
 
+<h3>Grouped</h3>
+<Chart::Bar
+  class="border"
+  @width="100%"
+  @height="200"
+  @variant="groupedLine"
+  @series={{this.seriesData}}
+  @noDataText="No data"
+  @valueAxisScale="shared"
```

Comment:
`@valueAxisScale` isn't needed for a `groupedLine` variant, since there's no way to have separate value axis scales if the lines are all being rendered on the same plot (or, at least, if you have more than 2 lines—and we're not supporting the 2-line dual-Y-axis version right now).

Reply:
Thanks for the feedback @fastfedora